common/tier4_state_rviz_plugin/src/autoware_state_panel.cpp

```diff
@@ -280,63 +147,349 @@ void AutowareStatePanel::onInitialize()
+
+  QVBoxLayout * groupLayout = new QVBoxLayout;
+  // set these widgets to show up at the left and not stretch more than needed
+  groupLayout->setAlignment(Qt::AlignCenter);
+  groupLayout->setContentsMargins(10, 0, 0, 0);
+  groupLayout->addWidget(group1);
+  // groupLayout->addSpacing(5);
+  groupLayout->addWidget(segmented_button, 0, Qt::AlignCenter);
+  return groupLayout;
+}
 
-    case OperationModeState::LOCAL:
-      text = "LOCAL";
-      style_sheet = "background-color: #FFFF00;";  // yellow
-      break;
+QVBoxLayout * AutowareStatePanel::makeRoutingGroup()
+{
+  auto * group = new QVBoxLayout;
 
-    case OperationModeState::REMOTE:
-      text = "REMOTE";
-      style_sheet = "background-color: #FFFF00;";  // yellow
-      break;
+  auto * custom_container = new CustomContainer(this);
 
-    case OperationModeState::STOP:
-      text = "STOP";
-      style_sheet = "background-color: #FFA500;";  // orange
-      break;
+  routing_icon = new CustomIconLabel(QColor("#84c2e6"));
 
-    default:
-      text = "UNKNOWN";
-      style_sheet = "background-color: #FF0000;";  // red
-      break;
-  }
+  clear_route_button_ptr_ = new CustomElevatedButton("Clear Route");
+  clear_route_button_ptr_->setCheckable(true);
+  clear_route_button_ptr_->setCursor(Qt::PointingHandCursor);
+  connect(clear_route_button_ptr_, SIGNAL(clicked()), SLOT(onClickClearRoute()));
 
-  if (msg->is_in_transition) {
-    text += "\n(TRANSITION)";
-  }
+  QLabel * routing_label = new QLabel("Routing");
+  routing_label->setStyleSheet("color: #d0e6f2; font-weight: bold;");
 
-  updateLabel(operation_mode_label_ptr_, text, style_sheet);
+  auto * horizontal_layout = new QHBoxLayout;
+  horizontal_layout->setSpacing(10);
+  horizontal_layout->setContentsMargins(0, 0, 0, 0);
 
-  // Control Mode
-  if (msg->is_autoware_control_enabled) {
-    updateLabel(control_mode_label_ptr_, "Enable", "background-color: #00FF00;");  // green
-  } else {
-    updateLabel(control_mode_label_ptr_, "Disable", "background-color: #FFFF00;");  // yellow
-  }
+  horizontal_layout->addWidget(routing_icon);
+  horizontal_layout->addWidget(routing_label);
+
+  custom_container->getLayout()->addLayout(horizontal_layout, 0, 0, 1, 1, Qt::AlignLeft);
+  custom_container->getLayout()->addWidget(clear_route_button_ptr_, 0, 2, 1, 4, Qt::AlignRight);
+
+  custom_container->setContentsMargins(10, 0, 0, 0);
+
+  group->addWidget(custom_container);
+
+  return group;
+}
+
+QVBoxLayout * AutowareStatePanel::makeLocalizationGroup()
+{
+  auto * group = new QVBoxLayout;
+  auto * custom_container = new CustomContainer(this);
+
+  init_by_gnss_button_ptr_ = new CustomElevatedButton("Initialize with GNSS");
+  init_by_gnss_button_ptr_->setCursor(Qt::PointingHandCursor);
+  connect(init_by_gnss_button_ptr_, SIGNAL(clicked()), SLOT(onClickInitByGnss()));
+
+  localization_icon = new CustomIconLabel(QColor("#84c2e6"));
+  QLabel * localization_label = new QLabel(" Localization");
```

Comment:
```suggestion
  QLabel * localization_label = new QLabel("Localization");
```

This will fix the alignment.